src/shared/Core.UI/Commands/CredentialsCommand.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Threading;
+using System.Threading.Tasks;
+using GitCredentialManager.UI.ViewModels;
+
+namespace GitCredentialManager.UI.Commands
+{
+    public abstract class CredentialsCommand : HelperCommand
+    {
+        protected CredentialsCommand(ICommandContext context)
+            : base(context, "basic", "Show basic authentication prompt.")
+        {
+            AddOption(
+                new Option<string>("--title", "Window title (optional).")
+            );
+
+            AddOption(
+                new Option<string>("--resource", "Resource name or URL (optional).")
+            );
+
+            AddOption(
+                new Option<string>("--username", "User name (optional).")
+            );
+
+            AddOption(
+                new Option("--no-logo", "Hide the Git Credential Manager logo and logotype.")
```

Comment:
Is there a reason this is `--no-logo` rather than `--logo`?

Reply:
I guess the default being 'show the logo'?